### ISSUE TITLE
Running zx throws error 'Windows Subsystem for Linux has no installed distributions.'

### ISSUE THREAD

**Issue by VelinGeorgiev** [google/zx#131]
I am playing with zx, but cannot seem to be able to make it work. Do I really need to install WSL to make it run on Windows?

## Expected Behavior

Runs a window cd command properly

## Actual Behavior

```
win32
C:\Users\xxx
$ cd 'C:\Users\xxx'
Windows Subsystem for Linux has no installed distributions.
Distributions can be installed by visiting the Microsoft Store:
https://aka.ms/wslstore
file:///C:/Users/xxx/AppData/Roaming/npm/node_modules/zx/index.mjs:52
        let output = new ProcessOutput({
                     ^

ProcessOutput [Error]:
    at file:///C:/p/zx-microsoft365cli/main.mjs:9:10
    at ChildProcess.<anonymous> (file:///C:/Users/xxx/AppData/Roaming/npm/node_modules/zx/index.mjs:52:22)
    at ChildProcess.emit (node:events:365:28)
    at maybeClose (node:internal/child_process:1067:16)
    at Socket.<anonymous> (node:internal/child_process:453:11)
    at Socket.emit (node:events:365:28)
    at Pipe.<anonymous> (node:net:661:12)
```

## Steps to Reproduce the Problem

Here is what I have
```
#!/usr/bin/env zx

void async function () {
  
  console.log(os.platform())
  console.log(os.homedir())

  await $`cd ${os.homedir()}`
}()
```

The moment I execute `zx  .\main.mjs` I get the above error.

## Specifications

- Version: 1.14.1
- Platform: Windows 10
- Nodejs: v16.2.0

**Comment by antonmedv** [comment]
It works for me in powershell and cmd.exe.

**Comment by VelinGeorgiev** [comment]
@antonmedv that is good for you, but it is not very helpful for me. I wouldn't call it great project support, when I have no additional clues to make it run other than it is running on someone else pc. So, thanks for the fish and I will not use your tool.

**Comment by antonmedv** [comment]
Let’s investigate what not working on some windows machines.

**Comment by antonmedv** [comment]
I’m close this issue. If you gave same problem, please create new issue.

**Comment by radiorz** [comment]
So the google/zx just use wsl actually but not powershell  so that I cannot use the global variables in powershell? such as I install  7zip in windows ,zx will tell me that it is not exist.

**Comment by chunhualiu** [comment]
> @antonmedv that is good for you, but it is not very helpful for me. I wouldn't call it great project support, when I have no additional clues to make it run other than it is running on someone else pc.

I met the same issue today.
After disabled WSL at control panel/programs/features, it worked.

**Comment by antonmedv** [comment]
There is PR with a fix: #458 
Will be merged after tests are added.
